app/components/floating-button/FloatingButton.js

```diff
@@ -37,6 +37,10 @@ class FloatingButton extends Component {
 
   handleClose = () => this.setState({ isShowingActions: false })
 
+  showComposer(type) {
+    return this.props.showComposer.bind(this, type)
```

Comment:
Can't you do something like: 
```js
constructor(props) {
    super(props);
    this.showComposer = ::this.props.showComposer
}
```
the `::` is the equivalent of `.bind(this)`, and the `type` arg is gonna be passed anyway, this is normally what I do in my react projects.